src/gui/qjacktrip.ui

```diff
@@ -146,7 +146,7 @@ To connect to a hub server you need to run as a hub client.</string>
        </size>
       </property>
       <property name="currentIndex">
-       <number>0</number>
+       <number>2</number>
```

Comment:
This should be 0 so that the app initially opens on to the first tab. (Qt Designer always saves the currently viewed tab as the default one. I've lost track of the number of times I've been stung by this myself.)

Reply:
How to edit this? Designer thinks it's set for 0 and manually editing the file isn't allowed.

Reply:
If you save it while you're viewing the first tab, that'll do the job.

Reply:
Thanks! That did the trick, fixed in latest commit